src/tbb/dynamic_link.cpp

```diff
@@ -379,8 +379,8 @@ namespace r1 {
     static dynamic_link_handle global_symbols_link( const char* library, const dynamic_link_descriptor descriptors[], std::size_t required ) {
         dynamic_link_handle library_handle{};
 #if _WIN32
-        auto res = GetModuleHandleEx(0, library, &library_handle);
-        __TBB_ASSERT_EX(res && library_handle || !res && !library_handle, nullptr);
+        bool res = GetModuleHandleEx(0, library, &library_handle);
```

Comment:
@longnguyen2004 seems like this change generates `warning C4800 'BOOL': forcing value to bool 'true' or 'false'`, that leads to error while compiling on VS2015

Reply:
I didn't remember making that change, but I'll change it back soon.